data/OR/incentives.json

```diff
@@ -0,0 +1,962 @@
+      "homeowner"
+    ],
+    "short_description": {
+      "en": "$1,000 rebate for a ductless heat pump. Must replace electric resistance as the primary heat source and primary unit must be in the main living space."
+    },
+    "start_date": "2023"
+  },
+  {
+    "id": "OR-9",
+    "authority_type": "utility",
+    "authority": "or-portland-general-electric",
+    "payment_methods": [
+      "rebate"
+    ],
+    "item": "rooftop_solar_installation",
+    "program": "or_portlandGeneralElectric_solarForYourHome",
+    "amount": {
+      "type": "dollar_amount",
+      "number": 400
+    },
+    "owner_status": [
+      "homeowner"
+    ],
+    "short_description": {
+      "en": "$400 rebate for Portland General Electric customers. Systems must be installed by an approved Energy Trust solar trade ally contractor."
+    }
+  },
+  {
+    "id": "OR-10",
+    "authority_type": "utility",
+    "authority": "or-pacific-power",
+    "payment_methods": [
+      "rebate"
+    ],
+    "item": "rooftop_solar_installation",
+    "program": "or_pacificPower_solarForYourHome",
+    "amount": {
+      "type": "dollar_amount",
+      "number": 450
+    },
+    "owner_status": [
+      "homeowner"
+    ],
+    "short_description": {
+      "en": "$450 rebate for Pacific Power customers. Systems must be installed by an approved Energy Trust solar trade ally contractor."
+    }
+  },
+  {
+    "id": "OR-11",
+    "authority_type": "utility",
+    "authority": "or-portland-general-electric",
+    "payment_methods": [
+      "rebate"
+    ],
+    "item": "battery_storage_installation",
+    "program": "or_portlandGeneralElectric_solarForYourHome",
+    "amount": {
+      "type": "dollars_per_unit",
+      "number": 250,
+      "unit": "kilowatt",
+      "maximum": 3000
+    },
+    "owner_status": [
+      "homeowner"
+    ],
+    "short_description": {
+      "en": "$250/kWh for battery storage. $3,000 maximum per home. Systems must be installed by an approved Energy Trust solar trade ally contractor."
+    }
+  },
+  {
+    "id": "OR-12",
+    "authority_type": "utility",
+    "authority": "or-pacific-power",
+    "payment_methods": [
+      "rebate"
+    ],
+    "item": "battery_storage_installation",
+    "program": "or_pacificPower_solarForYourHome",
+    "amount": {
+      "type": "dollars_per_unit",
+      "number": 250,
+      "unit": "kilowatt",
+      "maximum": 3000
+    },
+    "owner_status": [
+      "homeowner"
+    ],
+    "short_description": {
+      "en": "$250/kWh for battery storage. $3,000 maximum per home. Systems must be installed by an approved Energy Trust solar trade ally contractor."
+    }
+  },
+  {
+    "id": "OR-13",
+    "authority_type": "utility",
+    "authority": "or-portland-general-electric",
+    "payment_methods": [
+      "rebate"
+    ],
+    "item": "rooftop_solar_installation",
+    "program": "or_portlandGeneralElectric_solarWithinReach",
+    "amount": {
+      "type": "dollars_per_unit",
+      "number": 1,
```

Comment:
This should be 0.9. I think it's a formatting issue with the spreadsheet not having sufficient decimal points. Trying fix that so it shows up as 0.9 in the spreadsheet and then rerunning (you might have to wait a few minutes for it to update).